pkg/reconciler/broker/broker.go

```diff
@@ -53,6 +55,11 @@ import (
 	"knative.dev/eventing/pkg/reconciler/names"
 )
 
+const (
+	brokerIngressTLSSecretName = "mt-broker-ingress-server-tls"
```

Comment:
Go lint check is failing with:
```
GitHub Actions
/ style / Golang / Lint
G101: Potential hardcoded credentials (gosec)
```
We should try to disable this check for this case since is wrong